rust_src/admin/remacs-helpers-tests.el

```diff
@@ -0,0 +1,21 @@
+;;; Code:
+
+(ert-deftest transmute-type-test ()
```

Comment:
ert-deftest should be indented the same amount as defun.

If you execute `(require 'ert)` in your current Emacs instance, it should do the right thing :)

Reply:
I will give it a go.